src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/styles.ts

```diff
@@ -0,0 +1,60 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Run `Cmd + S` because Prettier (I think...) expects these to be separated by a line